scrapegraphai/nodes/generate_answer_node.py

```diff
@@ -99,44 +100,48 @@ def execute(self, state: dict) -> dict:
             template_merge_prompt = template_merge
 
         chains_dict = {}
+        answers = []
 
         # Use tqdm to add progress bar
         for i, chunk in enumerate(tqdm(doc, desc="Processing chunks", disable=not self.verbose)):
             if len(doc) == 1:
+                # No batching needed for single chunk
```

Comment:
why checking the document length inside the for-loop and not outside of it?

rather, the for-loop should be in place only if the document has multiple chunks